kits/production-database-release-planner/README.md

```
@@ -0,0 +1,155 @@
+
+## Setup
+
+### Prerequisites
+
+- Node.js 18+ and npm.
+- A Lamatic project with the `release-safety-pipeline` flow deployed (see [`agent.md`](agent.md) for the expected flow contract).
+
+### Steps
+
+1. From the repository root, move into the app directory:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)